Remotion/Web/Development.WebTesting/DownloadInfrastructure/DownloadedFileFinder.cs

```diff
@@ -247,5 +222,23 @@ private string GetPartialFile (IEnumerable<string> newFiles)
     {
       return newFiles.SingleOrDefault (file => file.EndsWith (_partialFileExtension));
     }
+
+    private bool PartialFileExists (IEnumerable<string> newFiles)
+    {
+      return GetPartialFile (newFiles) != null;
+    }
+
+    private bool PartialFileWasFoundBefore ()

```

Comment:
The name `PartialFileWasFoundBefore` is not ideal imo, got no better suggestion tho. I guess adding docs here would help.

Reply:
I would really like to avoid additional comments. I've renamed the method again, though. Take a look and tell me what you think